Snakefile

```diff
@@ -80,32 +102,6 @@ rule clean:
         shell("snakemake -j 1 run_all_scenarios --delete-all-output")
 
 
-rule run_tests:
-    output:
-        touch("tests.done"),
-    run:
-        import os
-
-        shell("snakemake --cores all build_test_configs")
-        directory = "test/tmp"  # assign directory
-        for filename in os.scandir(directory):  # iterate over files in that directory
-            if filename.is_file():
-                print(
-                    f"Running test: config name '{filename.name}'' and path name '{filename.path}'"
-                )
-                if "custom" in filename.name:
-                    shell("mkdir -p configs/scenarios")
-                    shell("cp {filename.path} configs/scenarios/config.custom.yaml")
-                    shell("snakemake --cores 1 run_all_scenarios --forceall")
-                if "monte" in filename.name:
-                    shell("cp {filename.path} config.yaml")
-                    shell("snakemake --cores all solve_all_networks_monte --forceall")
-                else:
-                    shell("cp {filename.path} config.yaml")
-                    shell("snakemake --cores all solve_all_networks --forceall")
-        print("Tests are successful.")
-
-
```

Comment:
Why is this rule deleted?

Reply:
it is replaced by the more secure and readable makefile